assets.js

```diff
@@ -74,11 +79,21 @@ export default [
     "symbol": "DOT",
     "trustwallet-uid": "c354"
   },
+  {
+    "caip-19": "cosmos:sentinelhub-2/slip44:118",
+    "symbol": "DVPN",
+    "trustwallet-uid": null
+  },
   {
     "caip-19": "eosio:aca376f206b8fc25a6ed44dbdc66547c/slip44:194",
     "symbol": "EOS",
     "trustwallet-uid": "c194"
   },
+  {
+    "caip-19": "cosmos:sifchain-1/slip44:118",
+    "symbol": "EROWAN",
```

Comment:
@xpt1x , `EROWAN` is the ERC20 token.  The cosmos-sdk native token, which corresponds to the caip-19 entry, is `ROWAN`.